app.js

```diff
@@ -2,6 +2,9 @@
 
 // links to the content section of the html and adds an event listener which invokes the handle clikc function when someone clicks within the content tag
 var content = document.getElementById('content');
+// var left = document.getElementById('left');
+// var center = document.getElementById('center');
+// var right = document.getElementById('right');
 content.addEventListener('click', handleClick);
 
 //constructor to save the images and keep track of clicks and show ups
```

Comment:
Nice work! 

A few comments on the function's name though: 
 * Don't put the word 'constructor' in the name of your function. You're trying to indicate that this is a constructor function with the name, which is smart, but see the next comment. 
 * Constructor functions are the one thing in javascript that get a capitalized first letter. For example, I might have named this function 'Image.' The capitalized name indicates that this isn't an image object, this is the template Image from which all images are made. 

Does that make sense?